packages/webview_flutter/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: webview_flutter
 description: A Flutter plugin that provides a WebView widget on Android and iOS.
-version: 0.1.0+1
+version: 0.2.0
```

Comment:
The version should be 0.1.1 as this isn't a breaking API change.

Reply:
Ah, I thought I needed to patch the minor version when adding a feature. I changed it now! 😄